idunn/places/address.py

```diff
@@ -0,0 +1,22 @@
+from .place import Place
+from idunn.api.utils import build_blocks, get_geom, build_address
+
+class Address(Place):
+    PLACE_TYPE = 'address'
+
+    @classmethod
+    def load_place(cls, es_place, lang, settings, verbosity):
+        raw_address = es_place.get('address') or {}
+        address = build_address(raw_address)
+
+        return cls(
+            id=es_place.get('id', ''),
+            name=es_place.get('name', ''),
+            local_name=es_place.get('name'),
+            class_name=es_place.get('poi_class'),
+            subclass_name=es_place.get('poi_subclass'),
+            geometry=get_geom(es_place, 'Point'),
```

Comment:
we shouldn't hard-code `Point`. And I think we could always return the whole geometry object for any type. (Even if a center is not computed for non-point geometry for now)